styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+  font-family: 'Roboto', Helvetica, sans-serif;
+}
+
+body {
+  background-color: #191765;
+  max-width: 375px;
+  background-image: url(images/Ellipse\ 18.svg);
+  background-repeat: no-repeat;
+  background-position: 0 47px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+#header_list {
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  align-items: center;
+}
+
+#company {
+  font-size: 18px;
+}
+
+#presentation {
+  display: flex;
+  margin-left: 24px;
+  margin-top: 60px;
+  margin-right: 24px;
+}
+
+#pres {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  max-width: 327px;
```

Comment:
Kindly remove the max-width in this class for we have already set a default width in the body class for the entire page. Avoid reapting yourself always use DRY(Don't Repeat yourself) techniques in CSS.